src/main/java/frc/robot/Constants.java

```diff
@@ -35,4 +38,34 @@ public static class Vision {
     public static final Matrix<N3, N1> kSingleTagStdDevs = VecBuilder.fill(4, 4, 8);
     public static final Matrix<N3, N1> kMultiTagStdDevs = VecBuilder.fill(0.5, 0.5, 1);
   }
-}
+
+  public static class ClimbConstants {
+
+    public static final int DEVICE_ID_CLIMB_MOTOR_1 = 60;
+    public static final int DEVICE_ID_CLIMB_MOTOR_2 = 61;
+  }
+
+  public static class AlgaeDudeConstants {
+
+  }
+
+  public static class IndexerConstants {
+    public static final int DEVICE_ID_BELT = 70;
+
+    public static final AngularVelocity INTAKE_VELOCITY = RadiansPerSecond.of(2 * Math.PI * 6);
+    public static final AngularVelocity SCOREL1_VELOCITY = RadiansPerSecond.of(2 * Math.PI * -6);
+    public static final AngularVelocity EJECT_VELOCITY = RadiansPerSecond.of(2 * Math.PI * -12);
+  }
+
+  public static class ArmConstants {
+
+  }
+
+  public static class EffectorConstants {
```

Comment:
We decided this subsystem would be the "Game Piece Manipulator Subsystem" because it's not only the end effector, it's also the algae wheel.